client/state/selectors/get-scheduled-publicize-share-action-time.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * @param {Object} state Global state tree
+ * @param {Number} siteId Site ID
+ * @param {Number} postId Post ID
+ * @return {String} time when newly scheduled share action will be published
```

Comment:
Looks like this could actually be `String` or `false`. Although we could change the `get` below to default to an empty string. Not sure which is preferable for a failure.